server/src/test/java/org/elasticsearch/index/mapper/NumberFieldMapperTests.java

```diff
@@ -336,12 +336,12 @@ protected <T> T compileOtherScript(Script script, ScriptContext<T> context) {
             }
 
             @Override
-            ScriptFactory emptyFieldScript() {
+            protected ScriptFactory emptyFieldScript() {
```

Comment:
This class wouldn't bee needed in the "server" module after the move I think. Small changes in visibility are okay if needed, please comment them if you have questions.